exercises/concept/meltdown-mitigation/conditionals.py

```diff
@@ -1,8 +1,11 @@
+""" Meltdown Mitigation exercise """
```

Comment:
Docstrings should not be space padded. The docstring summary line should terminate in a period.